components/common/action.go

```diff
@@ -8,4 +8,7 @@ type RequiredActions struct {
 	ResetGPU bool `json:"reset_gpu"`
```

Comment:
let us make this a list of RepaireActionType instead of a few boolean types. Now we should only have two types: 1. Reboot 2. RepairHardware (remove reset GPU. just use reboot)

Reply:
Done